docker_explorer/storage.py

```diff
@@ -79,36 +77,21 @@ def _MakeExtraVolumeCommands(self, container_object, mount_dir):
         be mounted.
     """
     extra_commands = []
+    mount_points = container_object.GetMountpoints()
     if self.docker_version == 1:
       # 'Volumes'
-      container_volumes = container_object.volumes
-      if container_volumes:
-        for mountpoint, storage in container_volumes.items():
-          mountpoint_ihp = mountpoint.lstrip(os.path.sep)
-          storage_ihp = storage.lstrip(os.path.sep)
-          storage_path = os.path.join(self.root_directory, storage_ihp)
-          volume_mountpoint = os.path.join(mount_dir, mountpoint_ihp)
-          extra_commands.append(
-              ['/bin/mount', '--bind', '-o', 'ro', storage_path,
-               volume_mountpoint]
-          )
+      for (source, destination) in mount_points:
```

Comment:
when you loop over tuples in explorer.py line 208, you don't use parentheses to indicate a tuple and here you do, please make it consistent

Reply:
done